Source/Scene/Points3DTileContent.js

```diff
@@ -148,19 +155,42 @@ define([
         }
     });
 
+    function createFeatures(content) {
```

Comment:
Sounds good to me.  Is this already implemented?

Also, please add a discussion of this to the `pnts` spec so users know how to create per-point vs. per-feature metadata.  Keep the discussion generic, not tied to the Cesium implementation, which basically just does not support AGI-level get/set for per-point metadata.